html/schematics.py

```diff
@@ -62,9 +62,41 @@ def getQualityData(conn, schematicID):
 	return qualityData
 
 # Get a list item for a component ingredient
-def getComponentLink(cn, objectPath, ingredientType):
+def getComponentLink(cn, objectPath, ingredientType, galaxy, schematic):
+	criteriaStr = ''
+
+	if galaxy.isdigit():
+		relevantGalaxy = galaxy if schematic.galaxy == 0 else schematic.galaxy
+		criteriaStr += f"""
+			AND tSchematic.galaxy IN(0, {relevantGalaxy})
+			  AND tSchematic.schematicID NOT IN(
+			    SELECT
+			      schematicID FROM tSchematicOverrides
+			    WHERE
+			      galaxyID = {relevantGalaxy})
+		"""
```

Comment:
Needs a conditional check for NGE servers as well. Perhaps try to use `dbShared.getBaseProfs/1`